addons/openebs/template/base/install.sh

```diff
@@ -45,12 +45,40 @@ function openebs() {
 function openebs_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
+            prompt_migrate_from_rook
```

Comment:
can we move this to `openebs_pre_init`? you will have to check that kubernetes is installed.